tool/get-dart/dart_sdk_archive/lib/src/version_selector.dart

```diff
@@ -181,8 +181,9 @@ class VersionSelector {
                 versionInfo.version < Version(3, 0, 0, pre: '290.2.beta')) {
               continue;
             }
-            // No stable builds yet.
-            if (versionInfo.channel == 'stable') {
+            // Stable builds start at 3.2.3.
+            if (versionInfo.channel == 'stable' &&
+                versionInfo.version < Version(3, 2, 3)) {
```

Comment:
Even if the builds existed at this point, would it instead make sense to only surface them in 3.3.0 and later? That way there's a more clear reference point on when they were declared stable. Generally anyone wanting to try on these platforms should use Dart 3.3 or later anyway.

I don't feel strongly either way. But curious to see what you both think?

Reply:
Updated to show from 3.3.